Index: source/src/renderer/AsyncResourceGatherer.hpp
===================================================================
--- source.orig/src/renderer/AsyncResourceGatherer.hpp
+++ source/src/renderer/AsyncResourceGatherer.hpp
@@ -12,6 +12,8 @@
 #include <hyprgraphics/cairo/CairoSurface.hpp>
 #include <hyprutils/os/FileDescriptor.hpp>
 
+#define CAIRO_FORMAT_RGB96F 6
+
 class CAsyncResourceGatherer {
   public:
     CAsyncResourceGatherer();
