Description: Disable check-licenses unit test.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: <not-required>, Debian-specific

--- a/test/meson.build
+++ b/test/meson.build
@@ -25,8 +25,8 @@
         ])
 endforeach
 
-check_licenses_script = files(meson.current_source_dir() + '/check-licenses.py')
-test('check-licenses', py, args: [check_licenses_script])
+#check_licenses_script = files(meson.current_source_dir() + '/check-licenses.py')
+#test('check-licenses', py, args: [check_licenses_script])
 
 check_tests_in_meson_script = files(meson.current_source_dir() + '/check-all-tests-are-in-meson.py')
 test('check-all-tests-are-in-meson', py, args: [check_tests_in_meson_script])
