Index: source/src/protocols/types/DMABuffer.cpp
===================================================================
--- source.orig/src/protocols/types/DMABuffer.cpp
+++ source/src/protocols/types/DMABuffer.cpp
@@ -108,7 +108,7 @@ CFileDescriptor CDMABuffer::exportSyncFi
     if (!good())
         return {};
 
-#if !defined(__linux__)
+#if 1
     return {};
 #else
     std::vector<CFileDescriptor> syncFds;
